SUPPORT.md

```diff
@@ -1,24 +1,22 @@
-# TODO: The maintainer of this repo has not yet edited this file
+# Support
 
-**REPO OWNER**: Do you want Customer Service & Support (CSS) support for this product/project?
+## How to file issues and get help  
 
-- **No CSS support:** Fill out this template with information about how to file issues and get help.
-- **Yes CSS support:** Fill out an intake form at [aka.ms/spot](https://aka.ms/spot). CSS will work with/help you to determine next steps. More details also available at [aka.ms/onboardsupport](https://aka.ms/onboardsupport).
-- **Not sure?** Fill out a SPOT intake as though the answer were "Yes". CSS will help you decide.
+* You can use [GitHub Issues](https://github.com/Azure/iot-hub-device-update/issues) to track bugs and feature requests. Please search the existing issues before filing new issues to avoid duplicates.
 
-*Then remove this first heading from this SUPPORT.MD file before publishing your repo.*
+* You can contact Microsoft Support, see details [here](https://docs.microsoft.com/azure/iot-hub-device-update/troubleshoot-device-update#contact)
 
-# Support
+## Contributing code or documentation
 
-## How to file issues and get help  
+We require pull-requests for code and documentation to be submitted against a new branch in order to review and run it in our gated build system. We try to maintain a high bar for code quality and maintainability, we insist on having tests associated with the code, and if necessary, additions/modifications to the requirement documents. Please include 'adu-client' as an approver. Also, have you signed the [Contribution License Agreement](https://cla.microsoft.com/) ([CLA](https://cla.microsoft.com/))? A friendly bot will remind you about it when you submit your pull-request.

```

Comment:
I've seen many projects on github that have CONTRIBUTING.md at the root of repo, which github has first-class support for.  If this were factored-out into CONTRIBUTING.md at top-level, then github will show a link to it when someone creates and issue or pull request, which could be helpful to potential contributors: https://github.blog/2012-09-17-contributing-guidelines/#contributors-wanted-inquire-within

Reply:
Added, thank you!